vscode/qodana/src/core/config/index.ts

```diff
@@ -33,29 +72,30 @@ class ConfigurationHelper {
     ];
 
     async resetSettings(context: vscode.ExtensionContext): Promise<void> {
-        await context.workspaceState.update('openedreport', null);
-        await context.workspaceState.update('reportId', null);
-        await context.workspaceState.update('computedPrefix', null);
-        await context.workspaceState.update('baselineIssues', false);
-        await context.globalState.update('javaExecutablePath', null);
+        await context.workspaceState.update(WS_OPENED_REPORT, null);
```

Comment:
Global state should be fully cleaned. 



there are 2 actions - reset auth state and reset all settings.
first one should reset auth info, the second one - all stored info. Seems like we are not resetting user info here.